laravel/app/Http/Controllers/MedhubController.php

```diff
@@ -16,263 +15,289 @@
     ...
  */
 
- // remodel is green 1611
+// remodel is green 1611
 
 namespace App\Http\Controllers;
 
-use Illuminate\Support\Facades\Log;
-use Illuminate\Http\Request;
-use Illuminate\Database\Eloquent\Model;
-use Illuminate\Support\Facades\DB;
 use GuzzleHttp\Client;
-use App\Resident;
-use App\EvaluateData;
-use App\Rotations;
-use App\Attending;
-use Mail;
-use \Datetime;
+use App\Models\Attending;
+use App\Models\Resident;
+use Illuminate\Support\Facades\Mail;
 
 class MedhubController extends Controller
 {
-
-	public function medhubConnect()
+    public function medhubConnect()
     {
         $users = self::activeResidentsPOST()->getBody(); //get string JSON
         $usersArr = json_decode($users, true); //turn json into an array
 
-		$fac = self::activeFacultyPOST()->getBody(); //get string JSON
+        $fac = self::activeFacultyPOST()->getBody(); //get string JSON
         $facArr = json_decode($fac, true); //turn json into an array
-		// echo var_dump($facArr);
+        // echo var_dump($facArr);
 
         $message = 'MedHub Active Residents';
 
-				$form = self::evaluationFormsPOST()->getBody();//get string JSON
+        $form = self::evaluationFormsPOST()->getBody(); //get string JSON
         $formArr = json_decode($form, true); //turn json into an array
-				$types = self::evaluationTypesPOST()->getBody();//get string JSON
+        $types = self::evaluationTypesPOST()->getBody(); //get string JSON
         $typesArr = json_decode($types, true); //turn json into an array
 
-        return view('schedules.admin.medhubTestPage', compact('message','usersArr', 'facArr', 'formArr', 'typesArr'));
-	}
-
+        return view('schedules.admin.medhubTestPage', compact('message', 'usersArr', 'facArr', 'formArr', 'typesArr'));
+    }
 
     // request format: json_encode(array('programID' => 73));
     // or just a JSON encoded string: '{"programID":73}'
     public function medhubPOST($callPath, $request = '{"programID":73}')
     {
         $client = new Client([
-            'base_uri' => 'https://osu.medhub.com/functions/api/'
+            'base_uri' => 'https://osu.medhub.com/functions/api/',
         ]);
-        $clientID='5006';
-        $privateKey='331xyg1hl65o';
+        $clientID = '5006';
+        $privateKey = '331xyg1hl65o';
 
         return $client->request('POST', $callPath, [
             'form_params' => [
-                $time = time(),
+                ($time = time()),
```

Comment:
first case of this change that I've seen from prettier, hopefully the logic doesn't change

Reply:
if I had to wager a guess, `$time` should actually be `'time'`.